v0/destinations/posthog/data/PHPropertiesConfig.json

```diff
@@ -140,5 +127,41 @@
     "destKey": "$app_build",
     "sourceKeys": "context.app.build",
     "isMobile": true
+  },
+  {
+    "destKey": "$viewport_height",
+    "sourceKeys": "context.viewport.height"
```

Comment:
There is no field called `context.viewport`. The height of the user's actual browser window (in pixels). We can map it from `traits`, or from `context.page`. WDYT @utsabc?

Reply:
updated mapping. mapped it with traits